packages/webapp/src/webview/state/middleware.ts

```diff
@@ -73,7 +75,9 @@ const allowedTelemetryActions = new Set([
     UPDATE_GUIDED_ANSWER_TREES,
     SHARE_LINK_TELEMETRY,
     OPEN_LINK_TELEMETRY,
-    RESET_FILTERS
+    RESET_FILTERS,
```

Comment:
Action `SYNCHRONIZE_BOOKMARK` is always called when a bookmark was clicked. It is used to asynchronously check if a bookmark needs to be updated. For telemetry we could use this and map it to the requirement 
> 4. When a Bookmarked guide is clicked

from ticket https://github.com/SAP/guided-answers-extension/issues/509